dd-sdk-android/src/main/kotlin/com/datadog/android/DatadogInterceptor.kt

```diff
@@ -198,7 +198,7 @@ internal constructor(
         val mimeType = response?.header(HEADER_CT)
         val kind = when {
             method in xhrMethods -> RumResourceKind.XHR
-            mimeType == null -> RumResourceKind.UNKNOWN
+            mimeType == null -> RumResourceKind.XHR
```

Comment:
maybe we need to add a comment why 2 branches have the same return value, just to avoid confusion for other devs who may think it is copy-paste error.